lib/osf-components/addon/components/node-card/template.hbs

```diff
@@ -80,7 +80,9 @@
                         {{/if}}
                     {{/if}}
 
-                    {{node-card/node-icon category=@node.category}}
+                    {{#if @node.category}}
+                        {{node-card/node-icon category=@node.category}}
+                    {{/if}}
```

Comment:
This doesn't look right. Don't we want to hide the category always for registrations, not just when there's no category set?